src/macro_expansion.jl

```diff
@@ -79,13 +79,21 @@ struct MacroExpansionError
     ex::SyntaxTree
     msg::String
     position::Symbol
+    stacktrace::Vector{Base.StackTraces.StackFrame}
 end
 
 """
 `position` - the source position relative to the node - may be `:begin` or `:end` or `:all`
 """
 function MacroExpansionError(ex::SyntaxTree, msg::AbstractString; position=:all)
-    MacroExpansionError(nothing, ex, msg, position)
+    MacroExpansionError(nothing, ex, msg, position, scrub_expand_macro_stacktrace(stacktrace(backtrace())))
+end
+
+function scrub_expand_macro_stacktrace(stacktrace::Vector{Base.StackTraces.StackFrame})
+    idx = @something findfirst(stacktrace) do stackframe::Base.StackTraces.StackFrame
+        stackframe.func === :expand_macro && stackframe.file === Symbol(@__FILE__)
+    end return stacktrace
```

Comment:
```suggestion
    end
```

Curious how this happened :sweat_smile: